FirebasePerformance/Sources/FPRClient.m

```diff
@@ -227,6 +253,15 @@ - (void)processAndLogEvent:(FPRMSGPerfMetric *)event {
     return;
   }
 
+  if (self.isFirstTimeProcessAndLogEvent) {
+    FPRLogInfo(
+        kFPRClientMetricLogged,
+        @"Welcome to Firebase Performance Monitoring! Please visit %@ in a minute for details.",
```

Comment:
This log message requires tweaking and we can align on that with Android.

Reply:
Sounds good. Will discuss this in the Android PR. https://github.com/firebase/firebase-android-sdk/pull/2632#discussion_r627804226

Reply:
Updated the logging message.